### DISCUSSION TITLE
알림유무 종류논의

### DISCUSSION THREAD

**Discussion by KimDaeSeong8721** [DeveloperAcademy-POSTECH/toGather#86] [General]
처음에 진행할 때는 알림유무를 하나로 생각하고 isAlarmOn 변수 하나를 만들었습니다.
이후 설정창, 친구저축현황이 추가되면서 친구 알림유무가 생겼고,  친구1 알림유무, 친구2 알림유무, 친구3 알림유무로 세분화되었습니다.
이 부분에 대해 논의가 필요합니다.
세 가지 방법 중에  하나를 선택해야 할거 같습니다.
1. isAlarmOn : Bool,하나로 통일
2. isAlarmOn : Bool, isFriendAlarmOn : Bool 
3. isAlarmOn : Bool,  areFriendsAlarmOn : [Bool]

**Comment by unuhqueen** [comment]
친구 한 명 한 명의 알람을 다르게 하는건 mvp에 포함되지 않는 것 같아서 저는 2번이 제일 나을 것 같습니다!

**Reply by yeongwooCho** [reply]
저도 각 친구의 알람 유무를 선택하는 것보다 
친구의 알람을 받는다 설정하면 모두 받는 것이 좋을 것 같습니다!
